android/app/build.gradle

```diff
@@ -43,7 +43,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.hometask_flutter_v2"
+        applicationId "com.example.hometask_v_3"
```

Comment:
version (v2, v3) shouldn't be a part of your app id (as your version will change regularly, while your package ID won't ever change after app is published)